tides/tide_fac.f

```diff
@@ -511,20 +512,32 @@ FUNCTION ANGLE(ARG)
 C   IF KEY EQ 0  NO QUADRANT SELECTION MADE
 C   IF KEY .NE. 0 PROPER QUADRANT IS SELECTED
 C---------------------------------------------------------------------
+      ! jgf: doing the minimum to eliminate the arithmetic if 
+      ! from this pile of spaghetti :-)
+      ! (the arithmetic if is a fortran deleted feature and 
+      ! its use produces warnings in gfortran 9.3.0)
       FUNCTION ARCTAN(TOP,BOTTOM,KEY)
       IF(BOTTOM .NE. 0.0) GO TO 4
-      IF(TOP) 2,9,3
+      IF (TOP.lt.0.) goto 2
+      if (top.eq.0.) goto 9
+      if (top.gt.0.) goto 3
     2 ARCTAN=270.
       RETURN
     3 ARCTAN=90.
       RETURN
     4 ARCTAN=ATAN(TOP/BOTTOM)*57.2957795
       IF(KEY.EQ.0) RETURN
-      IF(TOP) 5,5,7
-    5 IF(BOTTOM) 6,9,8
+      IF (TOP.lt.0.) goto 5
+      if (top.eq.0.) goto 5
```

Comment:
Is there a reason for the inconsistencies between the capitalized `IF` and lowercase `if` ?

Reply:
Yes I believe this change from arithmetic if to logical if makes the code more portable than before.

I changed the capitalization b/c I think it highlights that the code is new or that there was some discontinuity. The style change draws the eye to the code change. I am not wedded to that, if it makes more sense to maintain a consistent style then I can change it to all caps.